workers/docker-worker/src/lib/task.js

```diff
@@ -149,9 +150,12 @@ async function buildDeviceBindings(devices, expandedScopes) {
         },
       );
     });
+    if (device.binds) {
+      bindMounts = _.union(bindMounts, device.binds);
+    }
   }
 
-  return deviceBindings;
+  return [deviceBindings, bindMounts];
```

Comment:
Let's use a container object instead of a tuple, for clarity in `dockerConfig`

```suggestion
  return {deviceBindings, bindMounts};
```

Reply:
Thanks! I was not aware that you could omit the property name like this.